assets/components/productPage/list/unorderedList.jsx

```diff
@@ -14,14 +15,14 @@ type PropTypes = {|
 
 // ----- Render ----- //
 
-const ProductPageTextBlockList = ({
+const UnorderedList = ({
```

Comment:
Could also be a reusable component

Reply:
I wanna move everything from `productPage` out of it with time to be honest, should i just go for it in this PR?